web/src/UserListPage.js

```diff
@@ -116,6 +116,13 @@ class UserListPage extends React.Component {
   }
 
   renderTable(users) {
+    // transfer country code to name based on selected language
+    var countries = require("i18n-iso-countries");
```

Comment:
@seriouszyx can we use `import` instead of `require` ? Because `require` is usually Node.js's grammar. We better follow the ES6.

Same for the previous PR.

Reply:
I have considered. But the next line requires a dynamically constructed path based on the selected language. It seems [ES6 doesn't support it](https://stackoverflow.com/questions/30340005/importing-modules-using-es6-syntax-and-dynamic-path).

Reply:
Good points!